### PR TITLE
feat(resizable-view):  make handle more recognizable

### PR DESCRIPTION
**Pull request by Gaaaga**
### Description

Previous resize handles might be a bit hard to recognized with a dark background.
So, I changed the corner resize handles to SVG and did some changes to the other handles,gave the resize handles a better recognizable look :)

### Checklist

- [x] I have read the [**contributing**](https://github.com/remirror/remirror/blob/HEAD/docs/contributing.md) document.
- [x] My code follows the code style of this project and `pnpm fix` completed successfully.
- [x] I have updated the documentation where necessary.
- [x] New code is unit tested and all current tests pass when running `pnpm test`.

### Screenshots

https://user-images.githubusercontent.com/16329365/132666913-5d8739dc-712f-4bf7-8028-4298fbf117eb.mov

### CODE REVIEW

**Comment by ocavue** [comment]
@Gaaaga Could you rebase `origin/main`. There was a bug that causes Storybook not to work.

**Comment by Gaaaga** [comment]
@ocavue  fixed

File: .changeset/thin-planes-remain.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'prosemirror-resizable-view': minor
+---
+
+make handle more recognizable
```
Comment (ocavue) [inline .changeset/thin-planes-remain.md:5]:
```suggestion

Add a white border to the handle to make it more recognizable.
```

**Review by ocavue — APPROVED** [review]
LGTM

**Pull request merged**